engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -255,17 +255,21 @@ private void initRenderGraph() {
         BufferClearingNode reflectedRefractedClearingNode = new BufferClearingNode(reflectedRefractedBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
         renderGraph.addNode(reflectedRefractedClearingNode, "reflectedRefractedClearingNode");
 
-        FBOConfig sceneOpaqueFboConfig = displayResolutionDependentFBOs.getFboConfig(BUFFER1);
+        FBOConfig scenePrimaryBufferConfig = displayResolutionDependentFBOs.getFboConfig(BUFFER1);
+        FBOConfig sceneSecondaryBufferConfig = displayResolutionDependentFBOs.getFboConfig(BUFFER1);
 
-        BufferClearingNode readBufferClearingNode = new BufferClearingNode(sceneOpaqueFboConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
+        BufferClearingNode readBufferClearingNode = new BufferClearingNode(scenePrimaryBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
         renderGraph.addNode(readBufferClearingNode, "readBufferClearingNode");
 
+        BufferClearingNode writeBufferClearingNode = new BufferClearingNode(sceneSecondaryBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
```

Comment:
Clearing both BUFFER1 and BUFFER2 is essential (probably depth or stencil issues) as mentioned earlier, so shifted from a random line in a random Node to a proper Node.